lib/sps_king/account.rb

```diff
@@ -8,7 +8,8 @@ class Account
 
     attr_accessor :name,
                   :iban,
-                  :bic
+                  :bic,
+                  :schema_version
```

Comment:
Account should be about iban and bic and not the convenient tool to return the schema version